edb/schema/pointers.py

```diff
@@ -3139,8 +3139,9 @@ def get_or_create_union_pointer(
 
     far_endpoints = [p.get_far_endpoint(schema, direction)
                      for p in components]
-    targets: List[s_types.Type] = [p for p in far_endpoints
+    targets: Sequence[s_types.Type] = [p for p in far_endpoints
```

Comment:
Fix the indentation here. (flake8 would have caught this)

(Probably by dropping the iterator body to the next line.)